### ISSUE TITLE
Rename structure function error output column 

### ISSUE THREAD

**Issue by nevencaplar** [lincc-frameworks/tape#186]
At the moment, when calculating error, the output column is named `1_sigma`. That can be misleading, because 
1. It is calculated from quantile definition of the error (84 quantile - 16 quantile of the sample), which only correspond to 1_sigma if the underlying distribution is Gaussian
2. User can change at which quantile is the upper estimate and lower estimate taken - in which case this is really not `1_sigma`. 

I suggest to rename the column to sf2_err

**Comment by wilsonbb** [comment]
Hey Neven, @dougbrn mentioned that you might have started some work on this issue, and I wanted to see if it might still be a good first issue to pick up?

**Comment by nevencaplar** [comment]
Yes, it is a good first issue. My first/only immediate work here is the ticket #159 which I started before we discovered few bugs that were resolved in separate tickets.
